### DISCUSSION TITLE
Beginner question: replace Concentrator

### DISCUSSION THREAD

**Discussion by akeck** [Koenkk/zigbee2mqtt#22259] [General]
Hi,

I'm completely new to Z2M in Home Assistant and I love it :)

However, I'm currently using a SMLIGHT SLZB-06, but I want to replace it with a SMLIGHT SLZB-06**M** because of Matter support.
Now three beginner questions:
.) is it possible to use Zigbee and Matter together?
.) I have currently 40 devices on Zigbee. When I replace the concentrator, do I have to pair all over again or do I just have to set the new IP address in home assistant?
.) is it possible to use two concentrators in one infrastructure, just for redundancy?

I'm not only asking because the M model also supports Matter, but what is there to do if the concentrator fails some day...

thanks in advance :)

**Comment by francisp2** [comment]
Bad idea to use the same device for both Zigbee and Thread (Matter). Even the official comment from HA is to avoid it.
Better flash your SMLIGHT SLZB-06M with Thread only firmware.

> I have currently 40 devices on Zigbee. When I replace the concentrator, do I have to pair all over again or do I just have to set the new IP address in home assistant?

Yes, since it is another chipset, repairing required.

> is it possible to use two concentrators in one infrastructure, just for redundancy?

A Zigbee device can only be connected to one coordinator.

**Comment by akeck** [comment]
Although I don't like your answers, they helped me a lot. I was so afraid of it, but now at least I have the certainty.
Thanks for taking the time :)

Just one last question: I always try to have a device in reserve should one breaks. From what I understood of your answer, if I replace the current SLZB-06 with another of the same type, repairing is not necessary?

**Reply by francisp2** [reply]
Correct. A switch from one CC2652 to another CC2652 does not require repairing.
